holoviews/tests/plotting/bokeh/test_vectorfieldplot.py

```diff
@@ -83,3 +83,29 @@ def test_vectorfield_color_index_color_clash(self):
             "for 'line_color' option and declare a color_index; ignoring the color_index.\n"
         )
         self.assertEqual(log_msg, warning)
+
+    def test_vectorfield_no_hover_columns(self):
+        vectorfield = VectorField([(0, 0, 0), (0, 1, 1), (0, 2, 2)],
+                        vdims='color').opts(tools=[])
+        plot = bokeh_renderer.get_plot(vectorfield)
+        keys = plot.handles["cds"].data.keys()
+        assert len(keys) == 4
+        assert "x0" in keys
+        assert "y0" in keys
+        assert "x1" in keys
+        assert "y1" in keys
```

Comment:
```suggestion
    def test_vectorfield_no_hover_columns(self):
        vectorfield = VectorField([(0, 0, 0), (0, 1, 1), (0, 2, 2)],
                        vdims='color').opts(tools=[])
        plot = bokeh_renderer.get_plot(vectorfield)
        keys = sorted(plot.handles["cds"].data)
        assert keys == ["x0", "x1", "y0", "y1"]
```

It could also make these two functions a single parameterized test, but I don't have a strong opinion about that.

Reply:
I'd prefer not.

Reply:
Both?

Reply:
What's both?

Reply:
1) My code suggestion, with compacts the code
2) parameterize the two functions.

Reply:
Oh :) I applied 1